content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,440 @@
+
+In this blog post, I am excited to be able to introduce the Ansible playbooks for Data Services Cloud Console and show you how to use them. Along with the [Python SDK](https://github.com/HewlettPackard/greenlake-data-services-python) for Data Services Cloud Console, these playbooks should help you in your efforts to automate HPE GreenLake Data Services through an infrastructure-as-code approach.
+
+Ansible is an open-source IT automation tool that automates provisioning, configuration management, application deployment, and many other IT processes. 
+Two main features that make Ansible the best choice for automation are:
+
+* Ansible does not require any programming. 
+* Idempotence is offered as a built-in feature of many of the Ansible modules. This means the result of performing a task once is the same as performing it multiple times without any intervening actions.
+
+**Why do we need Ansible playbooks for Data Services Cloud Console?**
+
+Ansible helps the users/admins automate the deployment of resources and applications without the manual overhead of creating everything from scratch. These playbooks can be configured with conditions, variables, and tasks. Currently, simple playbooks, like performing CRUD operations on the resources, are available. These playbooks can be considered basic building blocks and can be reused to build simple-to-complex use cases.
+
+**[Ansible Modules for  Data Services Cloud Console](https://github.com/HewlettPackard/greenlake-data-services-ansible):**
+
+The following Ansible modules are currently available for Data Services Cloud Console. You can use the samples given or customize it - 
+
+*greenlake_audit_events_facts* - Get details of audit events
+
+*greenlake_host* - Manage host
+
+*greenlake_host_facts -* Get details of a host
+
+*greenlake_host_group -* Manage host group
+
+*greenlake_host_group_facts -* Get details of a host group
+
+*greenlake_host_initiator_facts -* Get details of a host initiator
+
+*greenlake_storage_system_facts -* Get details of a storage system
+
+*greenlake_volume -* Manage volume
+
+*greenlake_volume_facts -* Get details of a volume
+
+*greenlake_volumeset -* Manage volume set
+
+*greenlake_volumeset_facts -* Get details of a volume set
+
+**Prerequisites to use these Ansible playbooks:**
+
+1. Any machine with Python 3.8 or newer installed. This includes Red Hat, Debian, CentOS, macOS, any of the BSDs, and so on. (Microsoft Windows is not supported *)
+2. The latest and most stable version of Ansible must be installed. (current version is 2.9)
+3. HPE GreenLake Data Services Python SDK. (Installation procedure is mentioned below)
+4. Cloning the GitHub repo that has these playbooks
+5. Setup ANSIBLE_LIBRARY and ANSIBLE_MODULE_UTILS environment variables
```

Comment:
There's something odd about the display here for #5. 
![image](https://user-images.githubusercontent.com/77496261/228374013-65fa27b8-c3ef-4d6a-8f82-8ade7c569c14.png)

Reply:
Not sure. @choukden , can you please help with what is causing this?